_data/fellows.yml

```diff
@@ -1,11 +1,11 @@
 gabrielleBeaudry:
   name: Gabrielle Beaudry
-  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
+  info: 'BS Environmental Science, Energy Management track; Minor in Computer Science at UNC Chapel Hill'
```

Comment:
You might want to include her master's that she is starting:

BS Environmental Science, Energy Management track; Minor in Computer Science, Master's in Environmental Informatics Candidate at UNC Chapel Hill